src/modules/Dashboard/features/Participants/Participants.tsx

```diff
@@ -172,6 +172,11 @@ export const Participants = () => {
     nickname,
     tag,
   }: HandleUpgradeAccount) => {
+    Mixpanel.track('Upgrade to Full Account clicked', {
+      [MixpanelProps.AppletId]: appletId,
+      [MixpanelProps.Via]: 'Applet - Participants',
+    });
```

Comment:
Technically we can perform this action via the dashboard as well.

https://github.com/ChildMindInstitute/mindlogger-admin/blob/e6ff173fc6aaf31da3e81d92ab4f59caa8eef6da/src/modules/Dashboard/features/Respondents/Respondents.tsx#L162-L170

I wonder if we're interested in tracking those events as well. Is there a ticket for that already? If not, can you enquire about it?